jobs/ScheduleBackupJob.js

```diff
@@ -99,22 +130,43 @@ class ScheduleBackupJob extends BaseJob {
             .deleteBackup(deleteOptions)
             .then(() => backup.backup_guid);
         }
-      }).then(deletedBackupGuids => {
-        logger.info(`Successfully deleted backup guids : ${deletedBackupGuids} - instance deleted : ${instanceDeleted}`);
+      })
+      .then(deletedBackupGuids => {
+        // Deleting transaction logs from service-container.
+        let deletedTransactionLogFilesCount = 0;
+        return backupStore.listFilesOlderThan(listOptions, transactionLogsDeleteBefore, container)
```

Comment:
is this list function scalable for all IAASes ? Like for Azure we have limits of 5000 blobs , correct ?

Reply:
Yes, however, I think the understanding is that we don't have an use-case right now, for exceeding the number of blobs above 5000, as per the estimated current-usage.